src/Transport/Receiving/MessagePump.cs

```diff
@@ -272,28 +272,31 @@ await processor.CloseAsync(cancellationToken)
             // args.CancellationToken is currently not used because the v8 version that supports cancellation was designed
             // to not flip the cancellation token until the very last moment in time when the stop token is flipped.
             var contextBag = new ContextBag();
+            using var lockLostCancellationTokenSource = CancellationTokenSource.CreateLinkedTokenSource(messageProcessingCancellationToken);
+            CancellationToken lockLostCancellationToken = lockLostCancellationTokenSource.Token;
```

Comment:
why not var?

Reply:
Just to have more opportunities to asynchronously chat with you ;)